code/game/objects/items/storage/pouch.dm

```diff
@@ -598,6 +598,24 @@
 	new /obj/item/storage/box/combat_lolipop/tricord(src)
 	new /obj/item/storage/box/combat_lolipop/tramadol(src)
 
+/obj/item/storage/pouch/berrypouch
+	name = "berry bar pouch"
+	desc = "A pouch containing all of your berry needs."
+	icon_state = "barpouch"
+	sprite_slots = 3
+	storage_slots = 6
+	can_hold = list(
+		/obj/item/reagent_containers/food/snacks/wrapped/berrybar,
+	)
```

Comment:
```suggestion
	can_hold = list(/obj/item/reagent_containers/food/snacks/wrapped/berrybar)
```
instead of having a 1-item list take 3 lines. Don't add a trailing comma at the end though